vital_site/vital/views/security_view.py

```diff
@@ -63,11 +64,7 @@ def register(request):
                 user.sftp_account = user.email[:user.email.find('@')]
                 user.sftp_pass = user.password  # workaround to set sftp account
                 user.set_password(user.password)  # hashes the password
-<<<<<<< HEAD
-                activation_code = str(uuid.uuid4()).replace('-', '')
```

Comment:
Not necessary,needs to be fixed.

Reply:
No need to be fixed. Someone imported random lib and then replaced the way of generation of activation code. It should work fine.

Reply:
Someone might have made this change on local repo earlier and got pushed through my branch

Reply:
Might be, the change head seems to be master. Needs to be fixed. The activation code you get on production is 6 digit number now and not uuid. 
![image](https://user-images.githubusercontent.com/8259506/73243466-9b6e0000-41cd-11ea-9893-1ced0b8a942e.png)